### ISSUE TITLE
Add links to existing articles and videos published in 2019

### ISSUE THREAD

**Issue by valgrimm** [mdn/developer-portal#558]

**Comment by valgrimm** [comment]
Have counted and calculated. It will take about an 1.5 hours to add links to all of this content without images or author profiles. Images definitely needed, will need to define.
*Need to clarify if author profiles in scope.*

HACKS Posts
30 seconds to add without image
need featured images
adding all the authors will take extra time

11 posts october
7 september
3 august
6 July
6 June
8 May
7 April
8 March
7 February
8 January
71 posts 2019

71 posts x 30 seconds = 35.5 minutes

Layout Land videos from 2018, out of scope
Mozilla Hacks videos from 2019 = 21
Mozilla Developer videos = 18
total = 39 videos
If each video takes 1 minute to add that's 39 minutes

**Comment by valgrimm** [comment]
havi let me know articles have features images am waiting on Wordpress login to get these she contacted someone

**Comment by valgrimm** [comment]
I also contacted someone

**Comment by valgrimm** [comment]
adding links without images to get it done.
doing hacks first
october done already

**Comment by valgrimm** [comment]
Steve scraped images but then Jen provided links to relevant locations.

After referrals finally ended up with Ed Lim for access to Hacks Blog, Bugzilla ticket https://bugzilla.mozilla.org/show_bug.cgi?id=1594494

oh wait I just saw a notification I hve access
https://mail.google.com/mail/u/0/#inbox/FMfcgxwDsFdVClngjvjlCfFfdBdrxBjs

**Comment by valgrimm** [comment]
All videos from Mozilla Developer and Mozilla Hacks video channels have been added as video pages. External video links created have been removed along with their thumbnails. 
Now to add preview images to all Hacks posts.

**Comment by valgrimm** [comment]
All Hacks posts, Mozilla Developer videos, and Hacks videos are posted to the site. All Hacks posts have feature images. Videos do not need them as were added as pages.
Very happy this is done. 
Closing this ticket.

**Comment by valgrimm** [comment]
Videos added as pages need thumbnails, have discovered. However, creating them as pages made dev easier, apparently, so this was good. Will add thumbnails next week
